website_sale_suggest_create_account/__manifest__.py

```diff
@@ -4,11 +4,12 @@
 {
     "name": "Suggest to create user account when buying",
     "summary": "Suggest users to create an account when buying in the website",
-    "version": "10.0.1.1.0",
+    "version": "11.0.0.0.0",
     "category": "Website",
     "website": "https://www.tecnativa.com",
     "author": "Tecnativa, "
               "LasLabs, "
+              "Vauxoo, "
```

Comment:
Changes done doesn't deserve co-authorship IMO. If you would add tests...

Reply:
I'm not sure what is the OCA criteria to add a co-authorship.

Could you send us the guideline about this matter, please?

Reply:
@moylop260 There is no guideline yet and today this is open to interpretation.

My opinion is:

- Migration, bugfix, translation, documentation --> Contributors
- Development, Refactoring, Co-development --> Autors


-

Reply:
We can put in the middle migrations that requires serious efforts for making it work, but this one is not clearly one. That's why I say that if you add tests, then authorship won't be discussed.